### DISCUSSION TITLE
Margins for column.Item()

### DISCUSSION THREAD

**Discussion by zeyaddesigns** [QuestPDF/QuestPDF#780] [Q&A] [answered]
Hi there,

Is there a way I can add margins to a column.Item() within the Content container? So far I only see support for page margins, but I'm in a situation where I need to add some margins to text items within a column and have no way to do so.

Any help would be much appreciated.

Regards,
Ze

**Comment by girlpunk** [comment, marked as answer]
The [padding](https://www.questpdf.com/api-reference/padding.html) elements should do what you want

**Reply by zeyaddesigns** [reply]
Got it, thanks!

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
